week_2_workflow_orchestration/README.md

```diff
@@ -3,6 +3,9 @@
 > If you're looking for Airflow videos from the 2022 edition,
 > check the [2022 cohort folder](../cohorts/2022/week_2_data_ingestion/).
 
+> If you're looking for Prefect videos from the 2023 edition,
+> check the [2023 cohort folder](../cohorts/2023/week_2_workflow_orchestration/).
+
 Python code from videos is linked [below](#code-repository).
 
 Also, if you find the commands too small to view in Kalise's videos, here's the [transcript with code for the second Prefect video](https://github.com/discdiver/prefect-zoomcamp/tree/main/flows/01_start) and the [fifth Prefect video](https://github.com/discdiver/prefect-zoomcamp/tree/main/flows/03_deployments).
```

Comment:
I'd remove this too since it's prefect related

Reply:
This is done. Also removed the prefect links for homework and code repo